### PR TITLE
Add the map types for bpf_redirect_map()

### PR DESCRIPTION
**Pull request by lcp**
This patch series adds the new map types, DEVMAP and CPUMAP, for bpf_redirect_map() and updates the reference guide.

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, ok to test]

File: src/python/bcc/table.py [hunk 1]
```diff
@@ -766,3 +777,11 @@ def __len__(self):
 
     def clear(self):
         pass
+
+class DevMap(HashTable):
```
Comment (yonghong-song) [inline src/python/bcc/table.py]: Should DevMap be an Array instead of HashTable? The same for below CpuMap?

Comment (lcp) [reply src/python/bcc/table.py]: Right. They should use ArrayBase.

**Comment by lcp** [comment]
I'll update table.py and add an example for CPUMAP in v2.

**Pull request closed without merging**